library/entity/status_effect/src/main/java/org/quiltmc/qsl/entity/effect/mixin/LivingEntityMixin.java

```diff
@@ -41,8 +44,11 @@
 import org.quiltmc.qsl.entity.effect.api.StatusEffectRemovalReason;
 import org.quiltmc.qsl.entity.effect.api.StatusEffectUtils;
 import org.quiltmc.qsl.entity.effect.impl.QuiltStatusEffectInternals;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
-@Mixin(LivingEntity.class)
+// We want to make sure that our wrap operations are put before other mods, so that we wrap the vanilla call and not a mods call.
```

Comment:
```suggestion
// We want to make sure that our wrap operations are put before other mods, so that we wrap the vanilla call and not a mod's call.
```